### PR TITLE
feat: create login authentication

### PR DESCRIPTION
**Pull request by adzharamrullah**
## Changes

- feat: create login authentication

## Evidence
title: feat: create login authentication
project: Desa Digital
participants: @adzharamrullah @maruf12 @Ibwedagama @bangunbagustapa @naufalihsank @yoslie @maulanayuseph

### CODE REVIEW

File: components/Base/InputText/index.vue [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+        'input-text__wrapper-input': true,
+        'input-text__wrapper-input--focus': isFocused,
+        'input-text__wrapper-input--error': error
+      }"
+    >
+      <div
+        v-if="$slots['icon-left']"
+        :class="{
+          'input-text__icon-left': true,
+          'input-text__icon-left--border': iconLeftType === 'border' ? true : false,
```
Comment (naufalihsank) [inline components/Base/InputText/index.vue]: `input-text__icon-left--border': iconLeftType === 'border'` should be enough right ?

File: components/Base/InputText/index.vue [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+        autocomplete="on"
+        @focus="isFocused = true"
+        @blur="isFocused = false"
+        @input="onInput"
+      >
+      <div
+        v-if="type === 'password' || $slots['icon-right']"
+        :class="{
+          'input-text__icon-right': true,
+          'input-text__icon-right--border': iconRightType === 'border' ? true : false,
```
Comment (naufalihsank) [inline components/Base/InputText/index.vue]: this one too

File: components/Base/InputText/index.vue (hunk 1 above)
Comment (adzharamrullah) [reply components/Base/InputText/index.vue]: ah thanks you i forgot that sorry my bad

File: components/Base/InputText/index.vue (hunk 2 above)
Comment (adzharamrullah) [reply components/Base/InputText/index.vue]: yes thanks you mate

**Review by Ibwedagama — APPROVED** [review]

**Review by yoslie — APPROVED** [review]

**Review by maruf12 — APPROVED** [review]

**Review by bangun-jds — APPROVED** [review]

**Review by naufalihsank — APPROVED** [review]

**Review by maulanayuseph — APPROVED** [review]

**Pull request merged**